trunk/places/EG.php

```diff
@@ -130,21 +181,24 @@
     _x('Zefta', 'City of Egypt', 'states-cities-and-places-for-woocommerce'),
   ),
   'GIZ' => array(
-    _x('Abu Nomros', 'City of Egypt', 'states-cities-and-places-for-woocommerce'),
+    _x('Al Haram', 'City of Egypt', 'states-cities-and-places-for-woocommerce'),
     _x('Al Ayat', 'City of Egypt', 'states-cities-and-places-for-woocommerce'),
     _x('Al Badrasheen', 'City of Egypt', 'states-cities-and-places-for-woocommerce'),
     _x('Al-Bawaiti', 'City of Egypt', 'states-cities-and-places-for-woocommerce'),
     _x('Atfih', 'City of Egypt', 'states-cities-and-places-for-woocommerce'),
-    _x('Cheikh Zayed', 'City of Egypt', 'states-cities-and-places-for-woocommerce'),
+    _x('Agouza', 'City of Egypt', 'states-cities-and-places-for-woocommerce'),
+    _x('Dokki', 'City of Egypt', 'states-cities-and-places-for-woocommerce'),
+    _x('Sheikh Zayed', 'City of Egypt', 'states-cities-and-places-for-woocommerce'),
     _x('Giza', 'City of Egypt', 'states-cities-and-places-for-woocommerce'),
     _x('Hawamdiyah', 'City of Egypt', 'states-cities-and-places-for-woocommerce'),
     _x('Kafr Ghati', 'City of Egypt', 'states-cities-and-places-for-woocommerce'),
     _x('Kerdasa', 'City of Egypt', 'states-cities-and-places-for-woocommerce'),
-    _x('Manshiyet Al Bakari', 'City of Egypt', 'states-cities-and-places-for-woocommerce'),
+    _x('Manshiyet Al Bakari', 'City of Egypt', 'states-cities-and-places-for-woocommerce'), #In Cairo
```

Comment:
@HebaFareed could you please explain what means this comment (_In Cairo_)?

Reply:
sorry, this was an extra comment, please dismiss it